Sources/SyntaxKit/Result.swift

```diff
@@ -8,19 +8,23 @@
 //  Copyright © 2014-2015 Sam Soffes. All rights reserved.
 //
 
+import Foundation
+
 internal struct Result: Equatable {
 
     // MARK: - Properties
 
     let patternIdentifier: String
     var range: NSRange
+    let result: NSTextCheckingResult?
```

Comment:
Maybe rename this so that it is clear that this is the result of the match from a previous regex.